core/vision/ar_tag_detector.py

```diff
@@ -43,6 +43,17 @@ def is_ar_tag():
     return len(ar_tags) > 0
 
 
+def is_gate():
+    """
+    Returns whether there is an AR tag.
+
+    Returns:
+    -------------
+        detect (bool) - whether or not something was detected
+    """
```

Comment:
update docstring to reflect that we return if there is more than 1 ar tag. and maybe explain why we return true if there are 3, not just 2 (and then normally do a second check to abort out if there are more than 2)